tools/autograd/derivatives.yaml

```diff
@@ -1574,6 +1574,9 @@
 - name: to_sparse.sparse_dim(Tensor self, int sparse_dim) -> Tensor
   self: grad.to_dense()
 
+- name: to_sparse_csr(Tensor self) -> Tensor
```

Comment:
Why do you have this formula if you don't test it?

Reply:
I thought it would fix the issue before Ivan diagnosed it. I'm adding an explicit autograd test now.

Reply:
CSR layout is not yet supported by autograd. So the test won't pass. It's better to remove this formula for now, we'll add it later once autograd works.